code/modules/mob/living/brain/MMI.dm

```diff
@@ -24,6 +24,9 @@
 
 /obj/item/mmi/update_overlays()
 	. = ..()
+	. += add_mmi_overlay()
+
+/obj/item/mmi/proc/add_mmi_overlay()
 	if(brainmob && brainmob.stat != DEAD)
 		. += "mmi_alive"
```

Comment:
This proc is doing . += "string" but . is still null. This works but it's misleading.
```suggestion
		return "mmi_alive"
```